src/commands/game_options/remove.ts

```diff
@@ -206,24 +206,34 @@ export default class RemoveCommand implements BaseCommand {
             return;
         }
 
+        const remainingGroups = currentMatchedArtists.filter(
+            (group) => !matchedGroups.some((x) => x.id === group.id),
+        );
+
+        // Rematch the remaining groups to make sure collabs are not removed by accident.
```

Comment:
Is this re-matching still needed if we know which artists were added explicitly by the user?